app/src/main/java/com/framgia/fsalon/data/model/Scheduler.java

```diff
@@ -0,0 +1,51 @@
+package com.framgia.fsalon.data.model;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+
+import com.framgia.fsalon.BR;
+
+/**
+ * Created by beepi on 28/07/2017.
+ */
+public class Scheduler extends BaseObservable {
+    private String mNameCustomer;
+    private boolean mStatus;
+    private String mDate;
+
+    public Scheduler(String nameCustomer, boolean status, String date) {
+        mNameCustomer = nameCustomer;
+        mStatus = status;
+        mDate = date;
+    }
+
+    @Bindable
+    public String getNameCustomer() {
+        return mNameCustomer;
+    }
+
+    public void setNameCustomer(String nameCustomer) {
+        mNameCustomer = nameCustomer;
+        notifyPropertyChanged(BR.nameCustomer);
+    }
+
+    @Bindable
+    public boolean getStatus() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#BooleanGetMethodName